pkg/certificate/interfaces/interface.go

```diff
@@ -2,10 +2,23 @@ package certificate
 
 import (
 	"context"
+	"errors"
 	"net"
 	"time"
 )
 
+// ErrPending is returned when the Certificate is not in "Ready" state
+var ErrPending = errors.New("certificate creation pending")
+
+// ErrUnknown is returned when the Certificate status does not match the provider defined states
+var ErrUnknown = errors.New("certificate status unknown")
+
+// MaxRetries is the max number a function should be re-triggered in case of error
+const MaxRetries = 5
+
+// RetryInterval is the time to wait before the function is re-triggered
+const RetryInterval = time.Minute
```

Comment:
```suggestion
const (
// MaxRetries is the max number a function should be re-triggered in case of error
 MaxRetries = 5

// RetryInterval is the time to wait before the function is re-triggered RetryInterval = time.Minute
)
```